lib/src/api/method/mod.rs

```diff
@@ -659,7 +659,7 @@ where
 	/// let person: Option<Person> = db.select(("person", "h5wxrf2ewk8xjxosxtyc")).await?;
 	///
 	/// // You can skip an unnecessary option if you know the record already exists
-	/// let person: Option<Person> = db.select(("person", "h5wxrf2ewk8xjxosxtyc")).await?;
+	/// let person: Person = db.select(("person", "h5wxrf2ewk8xjxosxtyc")).await?;
```

Comment:
This used to be possible on beta 9 but it's no longer possible. We should delete this line instead, along with the comment.

Reply:
Done.